firedrills.md

```diff
@@ -31,6 +31,30 @@ node-1$ sudo systemctl status kube-apiserver
    Active: inactive (dead)
 ```
 
+#### Planet Container Properties
+
+Planet is a runc-based container that unshares the following namespaces:
```

Comment:
nit: `unshares` comes as a bit of too technical to me. I'd use something like `creates private copies of the following namespaces so it is able to manage resources independently of other processes`